Dockerfile

```diff
@@ -56,6 +56,17 @@ ADD ./get /home/ipython/py/get
 ENV PYTHONPATH /home/ipython/py/:$PYTHONPATH
 ENV PATH /home/ipython/py/:$PATH
 
+# ENV variables to replace conf file
+ENV DEBUG false
```

Comment:
Can you please put this in one command?
https://github.com/bgruening/docker-galaxy-stable/blob/master/galaxy/Dockerfile#L32